nominal/core/asset.py

```diff
@@ -210,6 +258,42 @@ def remove_data_sources(
         asset = self.__class__._from_conjure(self._clients, response)
         update_dataclass(self, asset, fields=self.__dataclass_fields__)
 
+    @deprecated("Use `remove_data_scopes` instead")
+    def remove_data_sources(
+        self,
+        *,
+        data_scope_names: Sequence[str] | None = None,
+        data_sources: Sequence[Connection | Dataset | LogSet | Video | str] | None = None,
+    ) -> None:
+        """Remove data sources from this asset.
+
+        The list data_sources can contain Connection, Dataset, Video instances, or rids as string.
+        """
+        self._remove_data_sources(data_scope_names=data_scope_names, data_sources=data_sources)
+
+    # Newer alias to replace `remove_data_sources`
+    def remove_data_scopes(
+        self,
+        *,
+        names: Sequence[str] | None = None,
+        rids: Sequence[str] | None = None,
+        scopes: ScopeSeq | None = None,
+    ) -> None:
+        """Remove data scopes from this asset.
+
+        Scopes can be specified either by their names, their rids, or the
+        scope objects themselves.
+        """
+        if isinstance(rids, str):
+            raise ValueError("Expected rids to be a sequence, not a string")
+
+        if rids is None:
+            rids = []
+        if scopes is None:
+            scopes = []
```

Comment:
I'd probably just keep this as one arg if they get splatted together anyways

Reply:
OK, merged